components/BusinessCard.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <v-card class="business-card">
+    <!-- City -->
+    <v-chip class="px-2 business-city">
+      <span>{{ city }}</span>
+    </v-chip>
+    <!-- Image -->
+    <div class="business-image">
+      <img
+        :alt="business.title + ' image'"
+        :src="
+          business.preview_image ||
+          require('@/static/images/fallback_preview.png')
+        "
+      />
+    </div>
+    <!-- Content -->
+    <div class="business-content card-content flex-column elevation-3">
+      <!-- Logo -->
+      <business-logo
+        :logo-info="{ logoFallback: business.title, logoPath: business.logo }"
```

Comment:
i would pass the whole business object here and calculate everything inside the component, or create individual props for src and alt